### PR TITLE
[MSSQL] Migration instructions

### PR DESCRIPTION
**Pull request by zucchinidev**
[#185202516](https://www.pivotaltracker.com/story/show/185202516)

Which other branches should this be merged with (if any)?

None

### CODE REVIEW

File: aws-rds-migration.html.md.erb [hunk 1]
```diff
@@ -12,8 +12,9 @@ the following:
   for AWS tile
 - PostgreSQL instance of the VMware Tanzu Service Broker for AWS tile
 - Aurora MySQL instance of the VMware Tanzu Service Broker for AWS tile
+- SQL Server instance of the VMware Tanzu Service Broker for AWS tile. Attention, the migration process is more restrictive and involves the mandatory use of [DMS](https://aws.amazon.com/dms/).
```
Comment (pivotal-marcela-campo) [inline aws-rds-migration.html.md.erb:15]: suggestion: Do we need to say there is no other choice? I would let customers decide that.

File: aws-rds-migration.html.md.erb [hunk 2]
```diff
@@ -64,13 +65,17 @@ After the migration is completed you can stop ongoing migration and, if you want
...
 For more detailed steps showing the Cloud Foundry commands necessary to use AWS DMS,
-see [Migrate Data from a Legacy MySQL Instance](#steps).
+see [Migrate Data from a Legacy Instance](#steps).
 
 ## <a id='steps'></a> Migrate Data from a Legacy RDS Instance
 
 To migrate data from an existing legacy RDS instance to <%= vars.product_short %>:
 
 1. Create a backup of the legacy RDS instance.
+   1. To create a backup in SQL Server, you will need a user with elevated privileges.
```
Comment (pivotal-marcela-campo) [inline aws-rds-migration.html.md.erb:75]: nitpick: the bullets here got a bit indented?

File: aws-rds-migration.html.md.erb [hunk 3]
```diff
@@ -64,13 +65,17 @@ After the migration is completed you can stop ongoing migration and, if you want
...
+   Credentials associated with applications that use this service do not have the necessary privileges to create a
```
Comment (fnaranjo-vmw) [inline aws-rds-migration.html.md.erb:76]: Is this correct? I thought the following change made every app effectively a db_owner with full permissions:
https://github.com/cloudfoundry/csb-brokerpak-aws/pull/1151/files#diff-f0ed80ac8022ae6c9695c1b88d3bbfe886b8c476ff9e0c289f484fba47d60ea6R18

Comment (pivotal-marcela-campo) [reply aws-rds-migration.html.md.erb:76]: This line is for the **legacy** broker. The change to db_owner is for the **csb** broker. Full priviledges are needed in both.

Comment (fnaranjo-vmw) [reply aws-rds-migration.html.md.erb:76]: Oh! I see it was modified in a subsequent PR.
https://github.com/pivotal-cf/docs-csb-aws/pull/310/files

I was just checking this PR to follow the instructions for migrating old instances.
I will follow instructions from main branch instead :)

**Pull request merged**